cmd/utils/flags.go

```diff
@@ -156,9 +156,9 @@ var (
 		Name:  "ethash.dagslockmmap",
 		Usage: "Lock memory maps for recent ethash mining DAGs",
 	}
-	SyncModeFlag = cli.StringFlag{
-		Name:  "syncmode",
-		Usage: `Default: "snap" for BSC, Mainnet and Goerli. "full" in all other cases`,
+	SnapshotFlag = cli.StringFlag{
```

Comment:
BoolFlag?

Reply:
I used StringFlag because if they don't specify `--snapshots` then the bool flag is false and vice versa, so we never get to use default on chains if not specified.

Reply:
But they can specify `--snapshots=false` when the flag is of type `BoolT`, no?

Reply:
I tried it with BoolFlag but it doesn't allow for defaulting chain. If the flag is not placed then it will automatically be false and if it is placed the true or we could do `--snapshots=false` to set it as false, but we cannot default as far as I know

Reply:
I mean `BoolT` instead of `Bool`

Reply:
`BoolTFlag`

Reply:
@enriavil1 there is method in cliCtx to check if flag is set or not.